### DISCUSSION TITLE
Sending syslog to sensor node

### DISCUSSION THREAD

**Discussion by stijnpieters** [Security-Onion-Solutions/securityonion#2148] [Unsupported Versions] [answered]
I have a distributed setup running SO2.3.10, consisting of 1 managersearch node and a couple of sensor nodes.

I'd like to send my syslogs to my sensor nodes, but nothing reaches my ELK stack. 
From what I've observed so far, syslogs sent to the sensor node are dropped by the firewall, disallowing the syslog logs to reach my filebeat container on the sensor.

Does anyone have syslog working on their sensor nodes?

**Comment by TOoSmOotH** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/firewall.html?highlight=syslog#allow-hosts-to-send-syslog-to-a-sensor-node
